docs/report/report.tex

```diff
@@ -45,6 +45,92 @@ \subsubsection{Stretch Goals}
 
 \section{Machbarkeitsstudie}
 
+\subsection{NFC Tag}
+\subsubsection{lesen}
+Um mit NFC Tags arbeiten zu können, müssen diese auch auf entschlüsselt bzw. gelesen werden können.
+Hierfür ist ein Hardware NFC-Reader notwendig, der über USB mit dem Chip der Box kommuniziert.
+Hierfür könnte \textit{evdev} genutzt werden, ein Kernel Modul von Linux. Um das mit Node zu nutzen, kann \textit{node-evdev}\footnote{https://github.com/sdumetz/node-evdev} genutzt werden. Mit dem Fork\footnote{https://github.com/anbraten/node-evdev} von Anbraten gibt es auch Raspberry Pi und Typescript support.
+Das ganze emuliert Keyboard Eingaben und sendet die Tag-IDs auf diese Art.
+
+\subsubsection{schreiben}
+Einen NFC-Tag zu beschreiben ist davon abhängig, ob der NFC-Tag über einen eigenen Speicher verfügt.
+Ist dieser vorhanden, können dort z.b. Kontaktdaten hinterlegt werden. Werden diese dann von einem Smartphone gelesen, öffnet sich die Kontakte-App und der auf dem NFC-Tag gespeicherte Kontakt kann abgespeichert werden.
+Dafür kann die gleiche Library wie zum lesen von NFC-Tags genutzt werden.
+
+\subsection{Raspberri Pi}
+\subsubsection{Docker Integration}
+Docker auf einem Raspberri Pi laufen zu lassen ist möglich und in vielen Anleitungen online nachlesbar.\footnote{https://phoenixnap.com/kb/docker-on-raspberry-pi}
+
+\subsubsection{Öffentlich zugängliches Web Interface}
+Um den Web Server vom Pi von außerhalb des Netzwerkes erreichen zu können, muss innerhalb des Routers der Port 80 ge-forwarded werden.
+Dann kann der Pi unter dessen öffentlicher IP xxx.xxx.xxx.xxx:80 erreicht werden.
+Da sich die öffentliche IP-Adresse eines privaten Internet-Anschlusses in der Regel täglich ändert, wird zum einfachen finden der IP ein DynDns Service benötig, welcher eine feste Domain in die wechselnde IP Adresse des Routers übersetzt.
+Alternativ ginge es auch ohne Port-Forwarding mit nginx und ngrok \footnote{https://vatsalyagoel.com/setting-up-a-public-web-server-using-a-raspberry-pi-3/}.
+
+\subsubsection{URL für UI festlegen}
+Über ein DNS Service, der auf dem Raspberri Pi läuft, wäre es möglich für die statische Public-IP eine eigene URL anzulegen.
+Dafür sind verschiedene DNS Services möglich, potentiell ist auch eine Domain notwendig.
+
+\subsection{User Interface}
+\subsubsection{Zugriff auf NFC Reader im Cloud Szenario}
+Tendenziell sollte es nicht notwendig sein, dass der Raspberri Pi, der das Web-UI hosted, auf den jeweiligen NFC-Reader der Box zugreifen muss.
+Sollte das doch der Fall sein, könnte der Chip der Box ein API Call an den Raspberri Pi mit den NFC-Tag Daten machen, um das jeweilige hinterlegte Event zu triggern.
```

Comment:
' Daten machen ' - auch diesen Abschnitt bitte einmal laut vorlesen, aber besten einer anderen Person und danach erneut schreiben.

Reply:
adressed in commit ebddf4dd3e0ad75eae259bd72ee957a80ea9e62b